### ISSUE TITLE
This sample is vulnerable and severely outdated

### ISSUE THREAD

**Issue by atifc** [oktadev/okta-dotnetcore-react-example#10]
Are there any plans to update this to current .NET version?

**Comment by mraible** [comment]
You're right. We should update it. I'll see if someone can write a new version.

**Comment by mraible** [comment]
If we create a new tutorial (that's similar to this one) that uses .NET 6 and React 18, would that satisfy your needs?

**Comment by atifc** [comment]
Apologies for the delay. Yes, this would be more than enough!
